compiler/testData/diagnostics/tests/multiplatform/headerClass/nestedClasses.kt

```diff
@@ -9,6 +9,8 @@ expect class OuterClass {
             class Another {
                 fun f(s: String)
                 val p: Int
+                val c : Int = <!EXPECTED_PROPERTY_INITIALIZER!>1<!>
+                val a: Int by <!EXPECTED_DELEGATED_PROPERTY!><!UNRESOLVED_REFERENCE, UNRESOLVED_REFERENCE!>lazy<!> { 1 }<!>
```

Comment:
Is the `UNRESOLVED_REFERENCE` here on `lazy` expected?

Reply:
Oh ok I guess you need to add `// WITH_RUNTIME` (?) or add something like
```kt
fun <T> lazy(initializer: () -> T): Lazy<T> = TODO()
interface Lazy<out T> {
    operator fun getValue(thisRef: Any?, property: KProperty<*>): T = TODO()
}
```
which I see in other tests

Reply:
`// WITH_RUNTIME` doesn't work; added definitions instead. I'd not bother with unresolved `TODO()`. :\